src/components/VideoCanvas/VideoCanvas.tsx

```diff
@@ -45,7 +45,7 @@ type VideoDimentions = {
   ratio: number,
 }
 
-const clampZoom = (value: number) => clamp(value, 0.25, 16);
+export const clampZoom = (value: number) => clamp(value, 0.1, 16);
```

Comment:
Can we get a constant variable to indicate what the 0.1 and 16 means - these numbers just appears some what arbitrary